build.tcl

```diff
@@ -101,6 +101,7 @@ proc findlib {path} {
 	    return $path
 	}
 	set new [file dirname $path]
+	puts [list hey $new]
```

Comment:
Is this an intentional change ? Or leftover debug output ?

Reply:
That would be leftover debug output.